src/definitions/modules/progress.js

```diff
@@ -23,6 +23,20 @@ window = (typeof window != 'undefined' && window.Math == Math)
     : Function('return this')()
 ;
 
+function toArray(element) {
```

Comment:
These functions need to be moved into the module because I'm pretty sure they will be global if they are placed here

Reply:
Done in https://github.com/fomantic/Fomantic-UI/pull/582/commits/1cf9677bb78449208231070fd6a25e122049fab9